src/frontend/src/flows/loginUnknown.ts

```diff
@@ -99,6 +100,14 @@ export const loginUnknown = async (
   });
 };
 
+function isRegistrationAllowed() {
+  return (
+    /^(https:\/\/identity\.ic0\.app$)|(https:\/\/[a-zA-Z0-9-]+\.[a-z]+\.dfinity\.network$)/.test(
```

Comment:
There is exactly one endpoint (namely `identity.raw.ic0.app`) that you want to disable registration on. Why not simply check for that?

Reply:
Done. AFAIK we have to check against 3 origins:
https://identity.raw.ic0.app, https://rdmx6-jaaaa-aaaaa-aaadq-cai.ic0.app, https://rdmx6-jaaaa-aaaaa-aaadq-cai.raw.ic0.app

Are you sure there are not more?

Reply:
I am not! I actually didn't even think of rdmx6-..., good catch! If we find more, then let's add them. Thanks!